docs/api-docs/catalog/v2-v3-examples.md

```diff
@@ -0,0 +1,1052 @@
+        "tiny_url": "https://cdn11.bigcommerce.com/{store_hash}/products/77/images/266/foglinenbeigestripetowel1b.1580767715.44.58.jpg?c=1"
+    },
+    "availability": "available",
+    "brand": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/brands/0",
+        "resource": "/brands/0"
+    },
+    "downloads": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/downloads",
+        "resource": "/products/77/downloads"
+    },
+    "images": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/images",
+        "resource": "/products/77/images"
+    },
+    "videos": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/videos",
+        "resource": "/products/77/videos"
+    },
+    "skus": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/skus",
+        "resource": "/products/77/skus"
+    },
+    "rules": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/rules",
+        "resource": "/products/77/rules"
+    },
+    "option_set": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/optionsets/14",
+        "resource": "/optionsets/14"
+    },
+    "options": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/options",
+        "resource": "/products/77/options"
+    },
+    ...
+    },
+    "metadata": []
+}
+```
+
+### Create a product
+
+To create a simple product using the V3 Catalog API, send a `POST` request to `/v3/catalog/products`.
+
+**V3 request**
+
+`POST /v3/catalog/products`
+
+```json
+{
+  "name": "BigCommerce Storage Basket",
+  "price": "10.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 4,
+  "type": "physical"
+}
+```
+
+To create a simple product using the V2 Products API, in addition to the properties in the V3 example, your `POST` request must include `availability`.
+
+**V2 request**
+
+`POST /v2/products`
+
+```json
+{
+  "name": "BigCommerce Clay Vase",
+  "price": "20.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 5,
+  "type": "physical",
+  "availability": "available"
+}
+```
+
+### Create a product with images
+
+To create a simple product with an image using the V3 Catalog API, first create the product, then add the image.
+
+**V3 request**
+
+`POST /v3/catalog/products/{product_id}/images`
+
+1. Create a product using `POST /v3/catalog/products`
+2. Add an image using `POST /v3/catalog/products/{product_id}/images`
+
+```json
+{
+  "is_thumbnail": true,
+  "sort_order": 1,
+  "description": "Top View",
+  "image_url": "https://upload.wikimedia.org/wikipedia/commons/7/7f/Anglel_Bless_Legendary_Hills_1_m%C4%9Bs%C3%ADc_st%C3%A1%C5%99%C3%AD.jpg"
+}
+```
+
+Creating a product with an image in V2 follows a similar sequence; however, the V2 `/images` endpoint only accepts the `multipart/form-data` media type.
```

Comment:
Will these directions work for both simple and complex products? Since you specify "simple" in the V3 directions, I recommend specifying here. Or if it doesn't matter whether it's simple or complex for either V2 or V3, perhaps referring to them both as "an image."